run.py

```diff
@@ -1011,6 +1011,16 @@ def main():
     # Sort alphabetically.
     tests.sort()
 
+    # If we don't find any tests, let's let the user know and exit
+    if not tests:
+        if args.exact:
+            print("Couldn't find any tests to match the pattern %s" % args.patterns)
+        elif args.testdir:
+            print("Couldn't find any tests to match the pattern %s" % args.testdir)
+        else:
+            print("Couldn't find any tests to match given pattern %s" % sys.argv[1])
+        return 1
```

Comment:
Perhaps, we could use improve error messages when handling extra non-mandatory flags.
e.g.
```
python ../suricata-verify/run.py -q jskefd
Couldn't find any tests to match given pattern -q
```

Reply:
more than right you are! merci!